example.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python3
+
+import boutvecma
+import easyvvuq as uq
+import chaospy
+import os
+import numpy as np
+import time
+
+campaign = uq.Campaign(name="Conduction")
```

Comment:
Worth adding a period at the end of the name to make the resulting folder name more readable?

Reply:
Wise! Or something else we can then add to the .gitignore?